core/dogu_v2.go

```diff
@@ -15,6 +23,23 @@ type Volume struct {
 	Owner       string
 	Group       string
 	NeedsBackup bool
+	Clients     []VolumeClient `json:"Clients,omitempty"`
+}
+
+// GetClient retrieves a client with a given name and return a pointer to it. If a client does not exist a nil pointer
+// and false are returned.
+func (v *Volume) GetClient(clientName string) (*VolumeClient, bool) {
+	if v.Clients == nil {
+		return nil, false
+	}
+
+	for i, _ := range v.Clients {
```

Comment:
🚫 **[checkstyle]** <gosimple> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>S1005: unnecessary assignment to the blank identifier